src/nccl_ofi_rdma.c

```diff
@@ -2309,10 +2317,12 @@ static inline int post_bounce_buffs(nccl_net_ofi_rdma_ep_t *ep)
 		}
 	}
 
-	ret = post_bounce_buffs_on_rail(ep, &ep->control_rail);
-	if (ret != 0) {
-		NCCL_OFI_WARN("Failed call to post_bounce_buffs_on_rail(control_rail)");
-		goto exit;
+	if (!ofi_nccl_rdma_rr_ctrl_msg()) {
```

Comment:
this patch doesn't do what we talked about.  I expected to see a patch that had num_rails control eps, but it looks like we're back to using the same endpoint for both data and control messages, which breaks Eric's work to use GPU buffers for eager bounce buffers.